### ISSUE TITLE
Merlin oscillating collimator: tidy up

### ISSUE THREAD

**Issue by DominicOram** [ISISComputingGroup/IBEX#3135]
As a user on MERLIN I would like the oscillating collimator to be a bit more friendly. As such do the following:
* The mounting radius, instrument scientists aren't sure what this means and what value to set it. If it's something the user should change it needs to be made clearer on what it is and it should be autosaved so that I don't lose it's value between cycles. If it does not need changing it should be saved as part of the configuration.
* The gmc code tells the IOC which mode it is using a string, as the IOC cannot read strings it never changes from initialising. The GMC should set a number instead.
* The collimator should not contribute to the motors are moving PV. If this is too hard pull out to separate ticket

**Comment by davidkeymer** [comment]
On the original VI for WISH, the mounting radius refers to the "encoder mounting radius" which is the distance from the pivot to the encoder and is therefore a fixed physical value.  No need for user (or anyone really) to change it.

**Comment by GDH-ISIS** [comment]
Sorry to add to the woes - on restarting the Galil ioc, the collimator appears to stop. To restart the collimator, it required changing the swept angle, then return back to the original value, then press start. Please check whether this is the case before starting any work relating to this.

**Comment by DominicOram** [comment]
Moving motors can be done as part of https://github.com/ISISComputingGroup/IBEX/issues/3159
